README.md

```diff
@@ -53,7 +51,7 @@ aws s3 --endpoint http://localhost:9878 cp --storage-class REDUCED_REDUNDANCY  /
 
 ### Run Ozone from released artifact
 
-If you need a more realistic cluster, you can [download](https://hadoop.apache.org/ozone/downloads/) the last (binary) release package, and start a cluster with the help of docker-compose:
+If you need a more realistic cluster, you can [download](https://ozone.apache.org/downloads/) the last (binary) release package, and start a cluster with the help of docker-compose:
```

Comment:
```suggestion
If you need a more realistic cluster, you can [download](https://ozone.apache.org/downloads/) the latest (binary) release package, and start a cluster with the help of docker-compose:
```

Not introduced in this patch, but it caught my eye.

Reply:
> Not introduced in this patch...

Not a problem and thanks the fix. I :heart: the boy-scout rule...

(OFF: which is also mentioned in this article what I read today: https://blog.rstankov.com/collaborative-single-player-mode)